tcms/telemetry/api.py

```diff
@@ -226,6 +226,42 @@ def test_case_health(query=None):
     return data
 
 
+@http_basic_auth_login_required
+@rpc_method(name="Testing.individual_test_case_health")
+def individual_test_case_health(query=None):
+
+    if query is None:
+        query = {}
+
+    res = {}
+    plan_id = 0
+    positive, negative, all_count = 0, 0, 0
+    for test_execution in TestExecution.objects.filter(**query).order_by(
+        "run__plan_id"
+    ):
+        status = test_execution.status
+
+        if status.weight > 0:
+            positive += 1
+        elif status.weight < 0:
+            negative += 1
+        all_count += 1
+
+        if test_execution.run.plan_id != plan_id:
+            plan_id = test_execution.run_id
+            res[plan_id] = {
+                "positive": positive,
+                "negative": negative,
+                "all": all_count,
+            }
+            positive, negative, all_count = 0, 0, 0
+
+    # append the last result
+    res[plan_id] = {"positive": positive, "negative": negative, "all": all_count}
```

Comment:
This data structure needs to expose the following attributes:

Completion rate == positive + negative statuses / all in % (neutral statuses aren't considered complete)
Failure rate == negative / completed in % (this is the opposite of stability rate shown in the images).

Whether or not it contains information about the individual positive/negative/neutral stats IDK. For now we don't need that.

Reply:
The mock-up in the blog post (https://kiwitcms.org/blog/kiwi-tcms-team/2019/03/03/legacy-reports-become-telemetry/) has a msg like `Failed 2 times in the last 4 runs`. If we only return the percents we won't be able to show that information. Is that ok? If yes, what would the text on the FE look like.

Reply:
Completion rate: 75 %
Failure rate: 30 % (of the completed 75)

Reply:
done. this is ready for review @atodorov

Reply:
Where are the completion rate/failure rate the previous comments talk about ?